src/web/app/components/sortable-table/sortable-table.component.ts

```diff
@@ -44,6 +53,12 @@ export class SortableTableComponent implements OnInit, OnChanges {
   // enum
   SortOrder: typeof SortOrder = SortOrder;
 
+  @Input()
+  tableId: string = '';
+
+  @Input()
+  headerColorScheme: SessionsTableHeaderColorScheme = SessionsTableHeaderColorScheme.BLUE;
```

Comment:
Should `SessionsTableHeaderColorScheme` be renamed and moved elsewhere? Feels weird to use a color scheme defined for Sessions Table for all `sortable-table`s

Reply:
Yup, sure! I have shifted it to the `sortable-table` component and updated the naming from `SessionsTableHeaderColorScheme` to `SortableTableHeaderColorScheme`.